prow/external-plugins/cherrypicker/lib/lib.go

```diff
@@ -19,10 +19,10 @@ package cherrypicker
 import "fmt"
 
 // CreateCherrypickBody creates the body of a cherrypick PR
-func CreateCherrypickBody(num int, requestor, note string) string {
+func CreateCherrypickBody(num int, requester, note string) string {
 	cherryPickBody := fmt.Sprintf("This is an automated cherry-pick of #%d", num)
-	if len(requestor) != 0 {
-		cherryPickBody = fmt.Sprintf("%s\n\n/assign %s", cherryPickBody, requestor)
+	if len(requester) != 0 {
+		cherryPickBody = fmt.Sprintf("%s\n\n/assign %s", cherryPickBody, requester)
```

Comment:
Please remove these, `requestor` is a valid variant of the [word](https://dictionary.cambridge.org/dictionary/english/requester) so this just pollutes the change.

/ok-to-test